librz/util/unum.c

```diff
@@ -913,3 +913,13 @@ RZ_API double rz_num_cos(double a) {
 RZ_API double rz_num_sin(double a) {
 	return sin(a);
 }
+
+RZ_API int rz_num_ffs(int a){
+        if (a == 0)
```

Comment:
1. Please use the clang-format
2. Please add the Doxygen `\brief` for all new API functions.